core/templates/dev/head/css/oppia.css

```diff
@@ -2247,6 +2459,100 @@ div#ng-curtain {
   z-index: 997;
 }
 
+.open > a {
```

Comment:
Have a comment delineating this section.

There is some navbar stuff earlier in this file. Shouldn't that be listed here instead?

Reply:
Done, all the nav-bar css are at one place which is overridden